packages/remix-azure-functions/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "@remix-run/azure-functions",
+  "description": "Azure Functions server request handler for Remix",
+  "version": "0.17.5",
```

We're way ahead now 🙈

```suggestion
  "version": "1.3.3",
```